Type confusion about assert package between agoric-sdk vs SES-shim

Actually, I'm not sure the diagnosis in the title is even accurate. The symptom is that `yarn lint` fails in the `SwingSet` package. It seems to be only SwingSet, which probably means the title diagnosis is wrong.

```
$ pwd
/Users/markmiller/src/ongithub/agoric/agoric-sdk/packages/SwingSet

$ yarn lint
yarn run v1.22.5
$ yarn lint:types&&yarn lint:eslint
$ tsc -p jsconfig.json
../../../SES-shim/packages/ses/src/error/types.js:1:1 - error TS6200: Definitions of the following identifiers conflict with those in another file: BaseAssert, AssertFail, AssertEqual, AssertTypeofBigint, AssertTypeofBoolean, AssertTypeofFunction, AssertTypeofNumber, AssertTypeofObject, AssertTypeofString, AssertTypeofSymbol, AssertTypeofUndefined, AssertTypeof, AssertString, AssertNote, DetailsToken, Details, StringablePayload, AssertQuote, DetailsTag, Assert

1 // @ts-check
  

  ../assert/src/types.js:1:1
    1 /* eslint-disable jsdoc/require-returns-check,jsdoc/valid-types */
      
    Conflicts are in this file.


Found 1 error.
```

--- a/packages/ERTP/jsconfig.json
+++ b/packages/ERTP/jsconfig.json
@@ -1,7 +1,7 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "esnext",
 
     "noEmit": true,
 /*

--- a/packages/SwingSet/jsconfig.json
+++ b/packages/SwingSet/jsconfig.json
@@ -1,7 +1,7 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "esnext",
 
     "noEmit": true,
 /*

--- a/packages/assert/jsconfig.json
+++ b/packages/assert/jsconfig.json
@@ -1,7 +1,7 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "esnext",
 
     "noEmit": true,
 /*

--- a/packages/cosmic-swingset/jsconfig.json
+++ b/packages/cosmic-swingset/jsconfig.json
@@ -1,7 +1,7 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "esnext",
 
     "noEmit": true,
 /*

--- a/packages/dapp-svelte-wallet/api/jsconfig.json
+++ b/packages/dapp-svelte-wallet/api/jsconfig.json
@@ -1,7 +1,7 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "esnext",
 
     "noEmit": true,
 /*

--- a/packages/deploy-script-support/jsconfig.json
+++ b/packages/deploy-script-support/jsconfig.json
@@ -1,7 +1,7 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "esnext",
 
     "noEmit": true,
 /*

--- a/packages/marshal/jsconfig.json
+++ b/packages/marshal/jsconfig.json
@@ -1,7 +1,7 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "esnext",
 
     "noEmit": true,
 /*

--- a/packages/notifier/jsconfig.json
+++ b/packages/notifier/jsconfig.json
@@ -1,7 +1,7 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "esnext",
 
     "noEmit": true,
 /*

--- a/packages/promise-kit/jsconfig.json
+++ b/packages/promise-kit/jsconfig.json
@@ -1,7 +1,7 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "esnext",
 
     "noEmit": true,
 /*

--- a/packages/store/jsconfig.json
+++ b/packages/store/jsconfig.json
@@ -1,7 +1,7 @@
 // This file can contain .js-specific Typescript compiler config.
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "esnext",
 
     "noEmit": true,
 /*

--- a/packages/store/src/types.js
+++ b/packages/store/src/types.js
@@ -1,3 +1,6 @@
+// eslint-disable-next-line spaced-comment
+/// <reference types="ses"/>
+
 /**
  * @typedef {Record<string, Function>} ExternalInstance
  */

